### PR TITLE
test: add test to test big collections or collections with big values

### PR DESCRIPTION
**Pull request by BorysTheDev**
fixes: #3931
In this test, I simulate replication for quite big containers with different numbers of keys and different key numbers

### CODE REVIEW

File: tests/dragonfly/replication_test.py [hunk 1]
```diff
@@ -2626,3 +2626,34 @@ async def test_replica_of_replica(df_factory):
...
+    c_master = master.client()
+    c_replica = replica.client()
+
+    logging.debug("Fill master with test data")
+    seeder = StaticSeeder(
+        key_target=20,
+        data_size=4000000,
+        collection_size=1000,
+        variance=100,
+        samples=1,
```
Comment (dranikpg) [inline tests/dragonfly/replication_test.py:2647]: samples is into how many sets we divide key_target to apply variance, maybe use > 1 to have mixed-size values

**Review by dranikpg — APPROVED** [review]
I'd add an opt_only case with values at least a few dozens of mbs

File: tests/dragonfly/replication_test.py (hunk 1 above)
Comment (BorysTheDev) [reply tests/dragonfly/replication_test.py:2647]: I wanted to have only one size, big or small or average maybe we can add one more test later to cover other cases

File: tests/dragonfly/replication_test.py [hunk 2]
```diff
@@ -2626,3 +2626,34 @@ async def test_replica_of_replica(df_factory):
         await c_replica2.execute_command(f"REPLICAOF localhost {replica.port}")
 
     assert await c_replica2.execute_command(f"REPLICAOF localhost {master.port}") == "OK"
+
+
+@pytest.mark.asyncio
+@pytest.mark.slow
+async def test_big_containers(df_factory):
```
Comment (adiholden) [inline tests/dragonfly/replication_test.py:2633]: I believe that this test should be one test case for test_replication_all in which you play different config params

File: tests/dragonfly/replication_test.py [hunk 3]
```diff
@@ -2626,3 +2626,34 @@ async def test_replica_of_replica(df_factory):
...
+
+    df_factory.start_all([master, replica])
+    c_master = master.client()
+    c_replica = replica.client()
+
+    logging.debug("Fill master with test data")
+    seeder = StaticSeeder(
+        key_target=20,
+        data_size=4000000,
+        collection_size=1000,
```
Comment (adiholden) [inline tests/dragonfly/replication_test.py:2645]: 1. one test case for big collection with more than kMaxBlobLen = 4092 elements , to check the flow in rdb load for big containers, the size of elemets should be small in this test , testing #3760 
2. second test case for big values of size ~10K, we can have 2 or 3 items in the collection in this test , testing https://github.com/dragonflydb/dragonfly/pull/3914

**Pull request merged**